CONTRIBUTING.md

```diff
@@ -54,4 +56,4 @@ new code.
  - Each formal release corresponds to a branch which is of the form `vX.Y.Z` where `X.Y.Z` maps
    directly onto the `package.json` (NPM) version.
  - Releases are also tagged so they are present on the Releases page on Github.
- - Releases should always be signed by the maintainers key
\ No newline at end of file
+ - Releases should always be signed by the maintainers key
```

Comment:
"always" is a filler word, adding no additional meaning.
Either choose "should" or "must".